Game/MyProject/Game/SceneManagement/CameraFollowAnimator.cpp

```diff
@@ -0,0 +1,67 @@
+//
+//  CameraAnimator.cpp
+//  RacingGame
+//
+//  Created by Tamas Both - (p) on 07/05/2019.
+//
+
+#include "CameraFollowAnimator.hpp"
+
+#include "CameraSceneNode.hpp"
+#include "CarAnimator.hpp"
+
+CameraFollowAnimator::CameraFollowAnimator(const SceneNode* target, const IvVector2& camera_circle, bool needs_smoothing) :
+    _target(target), _camera_circle(camera_circle), _needs_smoothing(needs_smoothing) {
+        // setting default values
+        if (needs_smoothing) {
+            _min_distance = 10.;
```

Comment:
Just a small note. I appreciate the fact that you try to write the correct data type so there is no conversion from int to float, but when you write "10." that's actually a double literal, so the conversion still happens double->float. The correct way to write float literals is either "10.f" or "10.0f"